packages/terra-form-select/src/combobox/Menu.jsx

```diff
@@ -153,7 +154,6 @@ class Menu extends React.Component {
 
   componentDidUpdate() {
     this.updateNoResultsScreenReader();
-    this.scrollIntoView();
```

Comment:
so we do not need this call during searching of an item and then selecting it ? is it still working fine as before ?

Reply:
Yes the functionality remains the same.